packages/compiler-cli/ngcc/src/analysis/decoration_analyzer.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 import {ConstantPool} from '@angular/compiler';
+import {TemplateSourceRegistryImpl} from '@angular/compiler-cli/src/ngtsc/annotations/src/template_source_registry';
```

Comment:
drive-by comment: imports from ngcc to ngtsc should be relative and target the barrel for that area

```suggestion
import {TemplateSourceRegistryImpl} from '../../../src/ngtsc/annotations';
```

Reply:
thanks, I fixed the imports